client/landing/top-links.tsx

```diff
@@ -78,42 +97,111 @@ const HideWhenSmall = styled.span`
   }
 `
 
-const TopLinks = () => {
+const HideWhenElectron = styled.li`
+  visibility: ${IS_ELECTRON ? 'hidden' : 'visible'};
```

Comment:
Also honestly feels like it might be easier to just not render these things in that case?
```tsx
{IS_ELECTRON ? <>
  <Link href='/splash'>Home</Link>
  <Link href='/faq'>FAQ</Link>
  <Link href='/ladder'>Ladder</Link>
</> : // ...
}
```

Think you overcomplicated things with the CSS approach, the value can't change at runtime anyway (which is why the sizing one was done this way)

Reply:
Yeah, the only reason I used the hidden visibility is because I thought the top-links bar looked kind of weird with just 2 centered links, but maybe it's fine :d

Reply:
Well the `Spacer` element should deal with that already shouldn't it? Like that should take up whatever space is left to push things to the right still

Reply:
Oh you're right, I wasn't rendering the spacer either so the links just got centered.